git-link.el

```diff
@@ -269,7 +299,7 @@
 	  dirname
 	  commit))
 
-(defun git-link-gitorious (hostname dirname filename branch commit start end)
+(defun git-link-gitorious (hostname dirname filename _branch commit start _end)
```

Comment:
Yes, it's a convention, more than that it's the elisp compiler feature. On phone, but you should find this link in elisp manual, and in use at many places in emacs core.